src/pages/starter-kit/structure.md

```diff
@@ -45,7 +45,9 @@ The following diagram shows the directory structure of the starter kit project:
 |   |   |   |__ webhook
 |   |   |__ webhook
 |   |   |   |__ <action>
-|   |__ onboarding
+|   |__ scripts
+|   |   |__ commerce-event-subscribe
+|   |   |__ onboarding
```

Comment:
If you've changed the folder structure, there are likely several references to files in the `/onboarding/` directory that may need to be changed to the `/scripts/onboarding` directory.

Reply:
Ouch! I missed some of those references. I've updated them in 0bd9f2d

Reply:
@rojoangel was `/test/onboarding` also changed to `/test/data/onboarding`?

Reply:
Good catch. We moved the tests from `/test/onboarding` to `test/scripts`. I've updated the docs in this commit: e6e4232